server/src/main/java/com/tencent/rss/server/MultiStorageManager.java

```diff
@@ -157,6 +168,11 @@ public MultiStorageManager(ShuffleServerConf conf, String shuffleServerId) {
     this.shuffleExpiredTimeoutMs = shuffleExpiredTimeoutMs;
     this.maxShuffleSize = maxShuffleSize;
 
+    this.maxForceUploadExpireTimeS = (long) (pendingEventTimeoutS * (maxShuffleForceUploadTimeRatio / 100.0));
```

Comment:
(pendingEventTimeoutS *  maxShuffleForceUploadTimeRatio / 100.0); better?

Reply:
IMHO do division first to avoid potential data overflow should be better.

Reply:
OK